docs/source/installation/login.md

```diff
@@ -39,39 +39,41 @@ To add a QHub user from the web console for Keycloak, visit <https://myqhubsite.
 All QHub users will be part of the `qhub` realm (a realm is a distinct identity provider and set of users in Keycloak).
 > NOTE: The root user alone is a member of the `master` realm.
 
-The `qhub` realm should be selected by default.
+The `qhub` realm is selected by default, we strongly recommend leaving it as is.
 
 Click 'Users' along the left-hand side of the page.
 
 Click the 'Add user' button and you will see the new user form:
 
 ![Keycloak add user tab screenshot - new user form ](../images/keycloak_add_users.png)
 
-There are three fields, outlined above, which should be filled out. These are 'Username', 'Email', and 'Groups'.
+Please fill out the three fields, outlined above. These are 'Username', 'Email', and 'Groups'.
 
 Depending on the authentication provider selected ('password', 'GitHub' or 'Auth0'), the values entered into the 'Username' field will differ slightly. The following table outlines those differences:
 
 |   | Password  | GitHub  | Auth0   |
 |---|---|---|---|
 | Username | *unique username*  | *GitHub username* | *Email to login with* |
 
-Once the 'Username' field has been updated, please add a valid email address in the 'Email' field.
-> NOTE: Although not required, users may not be able to log into Graphana if this field is not properly set.
+Once the 'Username' field is updated, please add a valid email address in the 'Email' field.
+> NOTE: Although not required, users may not be able to log into Grafana if this field isn't properly set.
 
 Lastly, associate the user with one or more of the 'Groups'. Out of the box, QHub is deployed with the following groups: 'admin', 'analyst', and 'developer' (see below for more information about 'Groups').
```

Comment:
Link directly to the section instead of see below.. e.g. see the Adding a new user section in this doc....